src/lib/components/wallets/Wallets.svelte

```diff
@@ -16,12 +16,8 @@
     const walletFactory = new WalletFactory();
     const dispatch = createEventDispatcher();
 
-    let filteredWallets: IWallet[];
-    let sortedWallets: IWallet[] = [];
-    if (wallets.length) {
-        filteredWallets = removeDuplicates(wallets).map(walletFactory.create);
-    } else {
-        filteredWallets = walletFactory.createAll();
+    async function createAllWalletsByDefault(): Promise<IWallet[]> {
```

Comment:
createWalletsByDefault. Is not necessary to clarify that we're gonna create `all` wallets because we're using the word `wallets` in plural and that already tells that will be more than one.